packages/destination-actions/src/destinations/talon-one/updateCustomerProfilesAttributes/index.ts

```diff
@@ -24,6 +24,50 @@ const action: ActionDefinition<Settings, Payload> = {
       label: 'Mutual Attribute-Value pairs',
       description:
         'This may contain mutual list of attributes and their values for every customer profile in the "data" array.'
+    },
+    attributesInfo: {
+      label: 'Attributes with types',
+      description: 'Use this field you want to specify an attribute type',
+      type: 'object',
+      required: false,
+      multiple: true,
```

Comment:
Ditto previous comments - beneficial to add a default path assuming customers will include the `attributesInfo` array in their identify call

```suggestion
      multiple: true,
      default: {
        '@arrayPath': [
          '$.traits.attributesInfo',
          {
            name: {
              '@path': '$.name'
            },
            type: {
              '@path': '$.type'
            }
          }
        ]
      },
```

Reply:
done